backend/realtime_worker.py

```diff
@@ -23,7 +26,8 @@ def fetch_tweets(db, stream):
         print ('Got tweet.')
         if isTransportOffer(tweet):
             print ('Adding tweet from @{}'.format(tweet['user']['screen_name']))
-            r.db('voyageavecmoi').table('offers').insert(get_tweet_data(tweet)).run(db)
+            tweet_data = retry_call(get_tweet_data, (tweet,), tries=MAX_TRIES)
```

Comment:
Hmm, good point, the issue was with `stream.statuses.filter(track='#VoyageAvecMoi')` actually (but I could not reproduce the error from Twitter so I couldn't test it)